module/mouse_gutbrain_de_independent_guide.qmd

```diff
@@ -143,17 +151,17 @@ We use the `library` command to load and attach packages to the R environment. T
 The "tidyverse" package that you loaded is useful for loading, wrangling, and exploring data.
 :::
 
-### Loading the gene expression count data 
+### Loading the gene expression count data
 
-The first analyses you might want to do involve exploring dataset that shows how much each gene is expressed overall. This is the gene expression count dataset. 
+The first analyses you might want to do involve exploring dataset that shows how much each gene is expressed overall. This is the gene expression count dataset.
 
 You can load this dataset into R using the following code:
 
 ```{r}
-gutbrain_genes <- read_csv("https://genomicseducation.org/data/mouse_gutbrain_de_total_counts.csv")
+gutbrain_genes <- read_csv("https://genomicseducation.org/data/mouse_gutbrain_de_counts.csv")
 ```
 
-This dataset has two columns: the first one is called `GeneID` and has the MGI gene IDs, while the second one is called `total_counts` and contains the total number of times each gene was expressed across all the samples.
+This dataset has 140 columns. The first one is called `GeneID` and has the MGI gene IDs. The next 138 have gene counts for each individual sample, while the last column is called `total_counts` and contains the total number of times each gene was expressed across all the samples.
```

Comment:
I found this phrasing a tad confusing. Maybe "each column is an individual mouse, except ..." ?